usaspending_api/awards/v2/filters/sub_award.py

```diff
@@ -90,8 +90,9 @@ def keyword_parse(keyword):
             logger.info('Found {} transactions based on keyword: {}'.format(len(transaction_ids), keyword))
             transaction_ids = [str(transaction_id) for transaction_id in transaction_ids]
             queryset = queryset.filter(latest_transaction_id__isnull=False)
-            queryset &= queryset.extra(
-                where=['"latest_transaction_id" = ANY(\'{{{}}}\'::int[])'.format(','.join(transaction_ids))])
+
+            sql_fragment = '"subaward_view"."latest_transaction_id" = ANY(\'{{{}}}\'::int[])'
```

Comment:
I'm not that experienced with Postgres, but `'{{{}}}'::int[]` seems like code that justifies an explanatory comment.

Reply:
The curly brackets are python string format syntax: https://docs.python.org/3.3/library/string.html#format-string-syntax

The inner `{}` is for the list of transaction ids, and the `{{` ..`}}` are for including a singe `{}` around the transaction ids in the final string.

I can add a comment